README.md

```diff
@@ -1,7 +1,65 @@
 # Lua-Modules
+
 ![Code Style](https://github.com/Liquipedia/LiquipediaMediaWikiMessages/workflows/Code%20Style/badge.svg)
 
-Used to keep versions of some important lua modules of the Liquipedia wiki that we want better version control for.
+Lua Modules are an integral part of Liquipedia. They allow for more complex logic for rendering elements. The website supports adding and editing these modules straight from the browser. For more complex modules however, a more integrated developer environment has been requested by several users. To provide in this need this github repository has been constructed.
+Any modules added in this repository will, after a review process, be added to the modules and templates of the website. Note that the modules in this repository are only a subset off those on the website. Any 'duplicates' on the website will be overwritten by those in this repository.
```

Comment:
technically only those with a "deploy header"
e.g. the following folders do not get deployed to the wikis:
- https://github.com/Liquipedia/Lua-Modules/tree/main/spec
- https://github.com/Liquipedia/Lua-Modules/tree/main/test_assets
- https://github.com/Liquipedia/Lua-Modules/tree/main/definitions
- https://github.com/Liquipedia/Lua-Modules/tree/main/plugins
- https://github.com/Liquipedia/Lua-Modules/tree/main/3rd